src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,21 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { ToDo } from '../../types';
+
+type Props = {
+  todos: ToDo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo
+          key={todo.id}
+          todo={todo}
+        />
+      ))}
+      ;
```

Comment:
There is an unnecessary semicolon after the TodoInfo component mapping. Removing this will clean up the code.

<img width="380" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/30219274/7791b4cc-4260-495e-9d02-3a17221d1bd1">